packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade/upgrade.routing.js

```diff
@@ -29,6 +31,15 @@ export default /* @ngInject */ ($stateProvider) => {
           (option1, option2) =>
             getOptionPrice(option1)?.value - getOptionPrice(option2)?.value,
         ),
+      renewPeriod: /* @ngInject */ (upgradeOptions, getOptionInterval) => {
+        let interval = DEFAULT_INTERVAL;
+        upgradeOptions.forEach((option) => {
+          if (getOptionInterval(option) !== DEFAULT_INTERVAL) {
+            interval = getOptionInterval(option);
```

Comment:
**nitpick:** `getOptionInterval` is called twice. Can we store it in a variable and avoid multiple calls?

Reply:
It's fixed.